### PR TITLE
bug: minor log fix

### PR DESCRIPTION
**Pull request by souravjain540**
### Description

wrong command in the log: 'pip install crawlee[playwright]', it should be "pip install 'crawlee[playwright]'"

### Issues

- N/A

### Testing

- N/A

### Checklist

- [ ] Changes are described in the `CHANGELOG.md`
- [ ] CI passed

### CODE REVIEW

**Comment by vdusek** [comment]
- I understand the reasoning behind using single quotes around the package name with the extra, as it helps with shell compatibility when dealing with brackets.
- The backticks... Since this is just a log line without any formatting, I'm unsure whether using backticks (`) isn't just less readable.

What do you think @janbuchar?

File: src/crawlee/beautifulsoup_crawler/__init__.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
     from .types import BeautifulSoupCrawlingContext
 except ImportError as exc:
     raise ImportError(
-        'To import anything from this subpacakge, you need to install the "beautifulsoup" extra. '
-        'For example, if you use pip, run "pip install crawlee[beautifulsoup]".',
+        "To import anything from this subpacakge, you need to install the 'beautifulsoup' extra."
```
Comment (janbuchar) [inline src/crawlee/beautifulsoup_crawler/__init__.py:6]:
```suggestion
        "To import anything from this subpackage, you need to install the 'beautifulsoup' extra."
```

File: src/crawlee/browsers/__init__.py [hunk 2]
```diff
@@ -4,8 +4,8 @@
     from .playwright_browser_plugin import PlaywrightBrowserPlugin
 except ImportError as exc:
     raise ImportError(
-        'To import anything from this subpacakge, you need to install the "playwright" extra. '
-        'For example, if you use pip, run "pip install crawlee[playwright]".',
+        "To import anything from this subpacakge, you need to install the 'playwright' extra."
```
Comment (janbuchar) [inline src/crawlee/browsers/__init__.py:7]:
```suggestion
        "To import anything from this subpackage, you need to install the 'playwright' extra."
```

File: src/crawlee/playwright_crawler/__init__.py [hunk 3]
```diff
@@ -3,8 +3,8 @@
     from .types import PlaywrightCrawlingContext
 except ImportError as exc:
     raise ImportError(
-        'To import anything from this subpacakge, you need to install the "playwright" extra. '
-        'For example, if you use pip, run "pip install crawlee[playwright]".',
+        "To import anything from this subpacakge, you need to install the 'playwright' extra."
```
Comment (janbuchar) [inline src/crawlee/playwright_crawler/__init__.py:6]:
```suggestion
        "To import anything from this subpackage, you need to install the 'playwright' extra."
```

**Review by janbuchar — APPROVED** [review]
@vdusek I think the backticks are fine. We may want to use colors instead at some point, but there's no need to stall such a minor PR because of that.

**Comment by B4nan** [comment]
hmm, we need the PR title check here. `bug:` prefix for any PR makes no sense unless you planned to introduce a new bug with the PR :D

**Comment by janbuchar** [comment]
> hmm, we need the PR title check here. `bug:` prefix for any PR makes no sense unless you planned to introduce a new bug with the PR :D 

I mean, we can update the log generator to put bug: in the same bucket as fix:

**Comment by souravjain540** [comment]
> hmm, we need the PR title check here. `bug:` prefix for any PR makes no sense unless you planned to introduce a new bug with the PR :D 

Hehehe, sorry! still used to the last company, there we were using "bug: for bug fixes", "feat: for new features" and similarly

**Comment by B4nan** [comment]
> I mean, we can update the log generator to put bug: in the same bucket as fix:

or we can actually follow the specs and not use random prefixes like `bug:` :D

**Comment by B4nan** [comment]
here is the PR check we use in other repositories

https://github.com/apify/crawlee/blob/master/.github/workflows/check-pr-title.yml

**Pull request merged**